sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs

```diff
@@ -0,0 +1,270 @@
+                            new PerpetualProperty($"Order/{testScope}/PortfolioManager", new PropertyValue("F Bar")),
+                            new PerpetualProperty($"Order/{testScope}/Account", new PropertyValue("J Wilson")),
+                            new PerpetualProperty($"Order/{testScope}/Strategy", new PropertyValue("UK Growth")),
+                        },
+                        lusidInstrument: "CCY_ZZZ" 
+                    ),
+                    new OrderRequest(
+                        code: orderId3,
+                        quantity: 300,
+                        instrumentIdentifiers: new Dictionary<string, string>()
+                        {
+                            ["Instrument/default/LusidInstrumentId"] = _instrumentIds.Skip(1).Take(1).Single()
+                        },
+                        portfolio: new ResourceId(testScope, "OrdersTestPortfolio"),
+                        properties: new List<PerpetualProperty>
+                        {
+                            new PerpetualProperty($"Order/{testScope}/TIF", new PropertyValue("GTC")),
+                            new PerpetualProperty($"Order/{testScope}/OrderBook", new PropertyValue("UK Test Orders 2")),
+                            new PerpetualProperty($"Order/{testScope}/PortfolioManager", new PropertyValue("F Bar")),
+                            new PerpetualProperty($"Order/{testScope}/Account", new PropertyValue("J Wilson")),
+                            new PerpetualProperty($"Order/{testScope}/Strategy", new PropertyValue("RiskArb")),
+                        },
+                        lusidInstrument: "CCY_ZZZ" 
+                    )
+                });
+
+            // We can ask the Orders API to upsert this order for us
+            var upsertResult = _ordersApi.UpsertOrders(scope: testScope, request: request);
+
+            // The return gives us a list of orders upserted, and the lusidinstrument for each has been mapped from the
+            // instrument identifiers passed
+            Assert.That(upsertResult.Values.Count(), Is.EqualTo(3));
+            Assert.That(upsertResult.Values.Single(rl => rl.Id.Id.Equals(orderId1)).LusidInstrument, Is.EqualTo(_instrumentIds.First()));
+            
+            var quantityFilter = _ordersApi.ListOrders(scope: testScope, asAt: DateTimeOffset.UtcNow, filter: "Quantity gt 100");
+
+            Assert.That(quantityFilter.Values.Count(), Is.EqualTo(2));
+            
+            var orderBookFilter = _ordersApi.ListOrders(scope: testScope, asAt: DateTimeOffset.UtcNow, filter: $"Properties[{testScope}/OrderBook] eq 'UK Test Orders 2'");
+
+            Assert.That(orderBookFilter.Values.Count(), Is.EqualTo(1));
+            
+            var instrumentFilter = _ordersApi.ListOrders(scope: testScope, asAt: DateTimeOffset.UtcNow, filter: $"InstrumentUid eq '{_instrumentIds.First()}'");
+            
+            Assert.That(instrumentFilter.Values.Count(), Is.EqualTo(2));
```

Comment:
Should we check the instrument id is correct?

Reply:
Can do.